pkgs/by-name/liberaforms/package.nix

```diff
@@ -0,0 +1,45 @@
+{
+  lib,
+  stdenv,
+  liberaformsEnv,
+  postgresql,
+  postgresqlTestHook,
+}:
+stdenv.mkDerivation {
+  pname = liberaformsEnv.name;
+  inherit (liberaformsEnv) version;
+  inherit (liberaformsEnv.config.mkDerivation) src dontConfigure installPhase;
+
+  doCheck = true;
+
+  nativeCheckInputs = [
+    liberaformsEnv.pyEnv
+    postgresql
+    postgresqlTestHook
+  ];
+
+  env = {
+    postgresqlEnableTCP = 1;
+    PGUSER = "db_user";
+  };
+
+  preCheck = ''
+    export LANG=C.UTF-8
+  '';
+
+  checkPhase = ''
+    runHook preCheck
+
+    # Run pytest on the installed version. A running postgres database server is needed.
+    (cd tests && cp test.ini.example test.ini && pytest -k "not test_save_smtp_config") #TODO why does this break?
```

Comment:
And pytestCheckHook here

Reply:
That would inherit an extra `pytest` (and probably `pytest-dotenv`) from Nixpkgs, which isn't very nice since all Python stuffs came from `liberaformsEnv.pyEnv`, generated by dream2nix.